Logging started at 2016-07-10 09:22:54
File name: C:\Users\Administrator\Dropbox\lunarisle source\.\data\logs\err\1468135374141 - Administrator - [Sun Jul 10][9.22AM].txt
System.getProperty("os.name"): "Windows Server 2012 R2"

[2016-07-10 09:39:57] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 09:39:57] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 09:39:57] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 09:39:57] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 09:39:57] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 09:39:57] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 09:39:57] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 09:39:57] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 09:40:00] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 09:40:00] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 09:40:00] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 09:40:00] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 09:40:00] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 09:40:00] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 09:40:00] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 09:40:00] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 10:29:25] java.lang.NullPointerException
[2016-07-10 10:29:25] 	at com.vencillio.rs2.entity.player.Player.login(Player.java:1208)
[2016-07-10 10:29:25] 	at com.vencillio.core.LoginThread.cycle(LoginThread.java:42)
[2016-07-10 10:29:25] 	at com.vencillio.core.LoginThread.run(LoginThread.java:83)
[2016-07-10 10:29:26] java.lang.NullPointerException
[2016-07-10 10:29:26] 	at com.vencillio.rs2.entity.player.Player.login(Player.java:1208)
[2016-07-10 10:29:26] 	at com.vencillio.core.LoginThread.cycle(LoginThread.java:42)
[2016-07-10 10:29:26] 	at com.vencillio.core.LoginThread.run(LoginThread.java:83)
[2016-07-10 16:05:01] java.lang.NullPointerException
[2016-07-10 16:05:01] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:01] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:01] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:97)
[2016-07-10 16:05:01] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:01] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:01] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:01] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:01] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:05:02] java.lang.NullPointerException
[2016-07-10 16:05:02] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:02] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:02] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:97)
[2016-07-10 16:05:02] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:02] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:02] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:02] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:02] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:05:04] java.lang.NullPointerException
[2016-07-10 16:05:04] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:04] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:04] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:213)
[2016-07-10 16:05:04] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:04] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:04] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:04] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:04] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:05:06] java.lang.NullPointerException
[2016-07-10 16:05:06] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:06] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:06] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:213)
[2016-07-10 16:05:06] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:06] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:06] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:06] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:06] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:05:07] java.lang.NullPointerException
[2016-07-10 16:05:07] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:07] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:07] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:213)
[2016-07-10 16:05:07] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:07] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:07] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:07] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:07] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:05:09] java.lang.NullPointerException
[2016-07-10 16:05:09] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:09] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:09] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:97)
[2016-07-10 16:05:09] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:09] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:09] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:09] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:09] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:05:10] java.lang.NullPointerException
[2016-07-10 16:05:10] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:10] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:10] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:213)
[2016-07-10 16:05:10] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:10] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:10] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:10] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:10] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:05:11] java.lang.NullPointerException
[2016-07-10 16:05:11] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:11] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:11] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:213)
[2016-07-10 16:05:11] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:11] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:11] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:11] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:11] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:05:13] java.lang.NullPointerException
[2016-07-10 16:05:13] 	at com.vencillio.rs2.entity.InterfaceManager.getParents(InterfaceManager.java:17)
[2016-07-10 16:05:13] 	at com.vencillio.rs2.entity.InterfaceManager.verify(InterfaceManager.java:89)
[2016-07-10 16:05:13] 	at com.vencillio.rs2.entity.player.net.in.impl.ItemPackets.handle(ItemPackets.java:97)
[2016-07-10 16:05:13] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:05:13] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:05:13] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:05:13] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:05:13] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 16:08:21] java.lang.NumberFormatException: For input string: "."
[2016-07-10 16:08:21] 	at java.lang.NumberFormatException.forInputString(Unknown Source)
[2016-07-10 16:08:21] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-10 16:08:21] 	at java.lang.Long.parseLong(Unknown Source)
[2016-07-10 16:08:21] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:208)
[2016-07-10 16:08:21] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 16:08:21] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 16:08:21] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 16:08:21] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 16:08:21] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 16:08:21] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 17:46:07] com.mashape.unirest.http.exceptions.UnirestException: java.net.SocketTimeoutException: Read timed out
[2016-07-10 17:46:07] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-10 17:46:07] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-10 17:46:07] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-10 17:46:07] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-10 17:46:07] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-10 17:46:07] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-10 17:46:07] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-10 17:46:07] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-10 17:46:07] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-10 17:46:07] 	at java.lang.Thread.run(Unknown Source)
[2016-07-10 17:46:07] Caused by: java.net.SocketTimeoutException: Read timed out
[2016-07-10 17:46:07] 	at java.net.SocketInputStream.socketRead0(Native Method)
[2016-07-10 17:46:07] 	at java.net.SocketInputStream.socketRead(Unknown Source)
[2016-07-10 17:46:07] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-10 17:46:07] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-10 17:46:07] 	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:136)
[2016-07-10 17:46:07] 	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:152)
[2016-07-10 17:46:07] 	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:270)
[2016-07-10 17:46:07] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
[2016-07-10 17:46:07] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
[2016-07-10 17:46:07] 	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:260)
[2016-07-10 17:46:07] 	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:161)
[2016-07-10 17:46:07] 	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:153)
[2016-07-10 17:46:07] 	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:271)
[2016-07-10 17:46:07] 	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:123)
[2016-07-10 17:46:07] 	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:254)
[2016-07-10 17:46:07] 	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
[2016-07-10 17:46:07] 	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
[2016-07-10 17:46:07] 	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
[2016-07-10 17:46:07] 	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
[2016-07-10 17:46:07] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
[2016-07-10 17:46:07] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:106)
[2016-07-10 17:46:07] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
[2016-07-10 17:46:07] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:138)
[2016-07-10 17:46:07] 	... 9 more
[2016-07-10 19:15:52] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 19:15:52] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-10 19:15:52] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-10 19:15:52] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-10 19:15:52] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-10 19:15:52] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-10 19:15:52] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-10 19:15:52] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-10 19:15:52] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-10 19:15:52] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-10 19:15:52] 	at java.lang.Thread.run(Unknown Source)
[2016-07-10 19:15:52] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 19:15:52] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-10 19:15:52] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-10 19:15:52] 	... 9 more
[2016-07-10 19:15:52] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 19:15:52] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-10 19:15:52] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-10 19:15:52] 	... 10 more
[2016-07-10 19:15:52] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 19:15:52] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-10 19:15:52] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-10 19:15:52] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-10 19:15:52] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-10 19:15:52] 	... 11 more
[2016-07-10 19:16:35] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 19:16:35] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-10 19:16:35] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-10 19:16:35] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-10 19:16:35] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-10 19:16:35] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-10 19:16:35] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-10 19:16:35] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-10 19:16:35] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-10 19:16:35] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-10 19:16:35] 	at java.lang.Thread.run(Unknown Source)
[2016-07-10 19:16:35] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 19:16:35] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-10 19:16:35] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-10 19:16:35] 	... 9 more
[2016-07-10 19:16:35] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 19:16:35] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-10 19:16:35] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-10 19:16:35] 	... 10 more
[2016-07-10 19:16:35] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 19:16:35] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-10 19:16:35] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-10 19:16:35] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-10 19:16:35] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-10 19:16:35] 	... 11 more
[2016-07-10 20:35:53] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 20:35:53] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-10 20:35:53] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-10 20:35:53] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-10 20:35:53] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-10 20:35:53] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-10 20:35:53] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-10 20:35:53] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-10 20:35:53] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-10 20:35:53] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-10 20:35:53] 	at java.lang.Thread.run(Unknown Source)
[2016-07-10 20:35:53] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 20:35:53] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-10 20:35:53] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-10 20:35:53] 	... 9 more
[2016-07-10 20:35:53] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 20:35:53] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-10 20:35:53] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-10 20:35:53] 	... 10 more
[2016-07-10 20:35:53] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 20:35:53] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-10 20:35:53] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-10 20:35:53] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-10 20:35:53] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-10 20:35:53] 	... 11 more
[2016-07-10 20:59:04] com.mashape.unirest.http.exceptions.UnirestException: java.net.SocketTimeoutException: Read timed out
[2016-07-10 20:59:04] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-10 20:59:04] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-10 20:59:04] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-10 20:59:04] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-10 20:59:04] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-10 20:59:04] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-10 20:59:04] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-10 20:59:04] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-10 20:59:04] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-10 20:59:04] 	at java.lang.Thread.run(Unknown Source)
[2016-07-10 20:59:04] Caused by: java.net.SocketTimeoutException: Read timed out
[2016-07-10 20:59:04] 	at java.net.SocketInputStream.socketRead0(Native Method)
[2016-07-10 20:59:04] 	at java.net.SocketInputStream.socketRead(Unknown Source)
[2016-07-10 20:59:04] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-10 20:59:04] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-10 20:59:04] 	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:136)
[2016-07-10 20:59:04] 	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:152)
[2016-07-10 20:59:04] 	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:270)
[2016-07-10 20:59:04] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
[2016-07-10 20:59:04] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
[2016-07-10 20:59:04] 	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:260)
[2016-07-10 20:59:04] 	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:161)
[2016-07-10 20:59:04] 	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:153)
[2016-07-10 20:59:04] 	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:271)
[2016-07-10 20:59:04] 	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:123)
[2016-07-10 20:59:04] 	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:254)
[2016-07-10 20:59:04] 	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
[2016-07-10 20:59:04] 	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
[2016-07-10 20:59:04] 	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
[2016-07-10 20:59:04] 	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
[2016-07-10 20:59:04] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
[2016-07-10 20:59:04] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:106)
[2016-07-10 20:59:04] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
[2016-07-10 20:59:04] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:138)
[2016-07-10 20:59:04] 	... 9 more
[2016-07-10 21:29:15] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:29:15] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:29:15] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:29:15] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:29:15] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:29:15] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:29:15] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:29:15] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:29:17] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:29:17] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:29:17] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:29:17] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:29:17] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:29:17] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:29:17] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:29:17] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:29:22] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:29:22] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:29:22] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:29:22] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:29:22] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:29:22] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:29:22] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:29:22] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:29:25] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:29:25] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:29:25] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:29:25] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:29:25] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:29:25] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:29:25] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:29:25] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:29:31] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:29:31] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:29:31] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:29:31] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:29:31] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:29:31] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:29:31] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:29:31] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:29:35] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:29:35] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:29:35] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:29:35] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:29:35] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:29:35] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:29:35] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:29:35] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:29:42] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:29:42] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:29:42] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:29:42] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:29:42] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:29:42] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:29:42] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:29:42] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:33:56] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:33:56] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:33:56] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:33:56] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:33:56] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:33:56] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:33:56] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:33:56] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:34:17] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:34:17] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:34:17] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:34:17] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:34:17] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:34:17] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:34:17] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:34:17] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:34:19] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:34:19] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:34:19] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:34:19] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:34:19] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:34:19] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:34:19] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:34:19] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:34:23] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:34:23] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:34:23] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:34:23] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:34:23] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:34:23] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:34:23] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:34:23] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:34:35] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:34:35] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:34:35] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:34:35] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:34:35] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:34:35] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:34:35] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:34:35] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:34:38] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:34:38] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:34:38] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:34:38] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:34:38] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:34:38] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:34:38] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:34:38] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:34:40] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:34:40] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:34:40] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:34:40] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:34:40] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:34:40] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:34:40] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:34:40] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:35:00] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:35:00] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:35:00] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:35:00] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:35:00] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:35:00] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:35:00] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:35:00] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:35:04] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:35:04] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:35:04] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:35:04] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:35:04] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:35:04] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:35:04] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:35:04] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:35:08] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:35:08] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:35:08] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:35:08] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:35:08] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:35:08] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:35:08] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:35:08] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:35:11] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:35:11] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:35:11] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:35:11] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:35:11] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:35:11] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:35:11] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:35:11] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:35:16] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:35:16] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:35:16] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:35:16] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:35:16] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:35:16] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:35:16] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:35:16] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:35:33] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:35:33] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:35:33] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:35:33] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:35:33] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:35:33] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:35:33] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:35:33] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:35:41] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:35:41] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:35:41] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:35:41] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:35:41] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:35:41] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:35:41] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:35:41] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:35:47] java.lang.ArrayIndexOutOfBoundsException: 998
[2016-07-10 21:35:47] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:35:47] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:35:47] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:35:47] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:35:47] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:35:47] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:35:47] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:38:47] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:38:47] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:38:47] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:38:47] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:38:47] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:38:47] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:38:47] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:38:47] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 21:38:53] java.lang.ArrayIndexOutOfBoundsException: 9998
[2016-07-10 21:38:53] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.handleCommand(PlayerCommand.java:335)
[2016-07-10 21:38:53] 	at com.vencillio.rs2.entity.player.net.in.impl.CommandPacket.handle(CommandPacket.java:54)
[2016-07-10 21:38:53] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 21:38:53] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 21:38:53] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 21:38:53] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 21:38:53] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 22:26:38] com.mashape.unirest.http.exceptions.UnirestException: java.net.SocketTimeoutException: Read timed out
[2016-07-10 22:26:38] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-10 22:26:38] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-10 22:26:38] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-10 22:26:38] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-10 22:26:38] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-10 22:26:38] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-10 22:26:38] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-10 22:26:38] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-10 22:26:38] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-10 22:26:38] 	at java.lang.Thread.run(Unknown Source)
[2016-07-10 22:26:38] Caused by: java.net.SocketTimeoutException: Read timed out
[2016-07-10 22:26:38] 	at java.net.SocketInputStream.socketRead0(Native Method)
[2016-07-10 22:26:38] 	at java.net.SocketInputStream.socketRead(Unknown Source)
[2016-07-10 22:26:38] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-10 22:26:38] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-10 22:26:38] 	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:136)
[2016-07-10 22:26:38] 	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:152)
[2016-07-10 22:26:38] 	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:270)
[2016-07-10 22:26:38] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
[2016-07-10 22:26:38] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
[2016-07-10 22:26:38] 	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:260)
[2016-07-10 22:26:38] 	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:161)
[2016-07-10 22:26:38] 	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:153)
[2016-07-10 22:26:38] 	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:271)
[2016-07-10 22:26:38] 	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:123)
[2016-07-10 22:26:38] 	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:254)
[2016-07-10 22:26:38] 	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
[2016-07-10 22:26:38] 	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
[2016-07-10 22:26:38] 	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
[2016-07-10 22:26:38] 	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
[2016-07-10 22:26:38] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
[2016-07-10 22:26:38] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:106)
[2016-07-10 22:26:38] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
[2016-07-10 22:26:38] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:138)
[2016-07-10 22:26:38] 	... 9 more
[2016-07-10 22:30:00] com.mashape.unirest.http.exceptions.UnirestException: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 22:30:00] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-10 22:30:00] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-10 22:30:00] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-10 22:30:00] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-10 22:30:00] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-10 22:30:00] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-10 22:30:00] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-10 22:30:00] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-10 22:30:00] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-10 22:30:00] 	at java.lang.Thread.run(Unknown Source)
[2016-07-10 22:30:00] Caused by: java.lang.RuntimeException: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 22:30:00] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:106)
[2016-07-10 22:30:00] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:139)
[2016-07-10 22:30:00] 	... 9 more
[2016-07-10 22:30:00] Caused by: java.lang.RuntimeException: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 22:30:00] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:51)
[2016-07-10 22:30:00] 	at com.mashape.unirest.http.HttpResponse.<init>(HttpResponse.java:95)
[2016-07-10 22:30:00] 	... 10 more
[2016-07-10 22:30:00] Caused by: org.json.JSONException: A JSONArray text must start with '[' at 1 [character 2 line 1]
[2016-07-10 22:30:00] 	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
[2016-07-10 22:30:00] 	at org.json.JSONArray.<init>(JSONArray.java:105)
[2016-07-10 22:30:00] 	at org.json.JSONArray.<init>(JSONArray.java:144)
[2016-07-10 22:30:00] 	at com.mashape.unirest.http.JsonNode.<init>(JsonNode.java:48)
[2016-07-10 22:30:00] 	... 11 more
[2016-07-10 22:31:37] com.mashape.unirest.http.exceptions.UnirestException: java.net.SocketTimeoutException: Read timed out
[2016-07-10 22:31:37] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143)
[2016-07-10 22:31:37] 	at com.mashape.unirest.request.BaseRequest.asJson(BaseRequest.java:68)
[2016-07-10 22:31:37] 	at com.motivoters.motivote.MotivoteService.callAPI(MotivoteService.java:37)
[2016-07-10 22:31:37] 	at com.motivoters.motivote.MotivoteService.redeemVote(MotivoteService.java:96)
[2016-07-10 22:31:37] 	at com.vencillio.rs2.entity.player.net.in.command.impl.PlayerCommand.lambda$0(PlayerCommand.java:64)
[2016-07-10 22:31:37] 	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[2016-07-10 22:31:37] 	at java.util.concurrent.FutureTask.run(Unknown Source)
[2016-07-10 22:31:37] 	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
[2016-07-10 22:31:37] 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
[2016-07-10 22:31:37] 	at java.lang.Thread.run(Unknown Source)
[2016-07-10 22:31:37] Caused by: java.net.SocketTimeoutException: Read timed out
[2016-07-10 22:31:37] 	at java.net.SocketInputStream.socketRead0(Native Method)
[2016-07-10 22:31:37] 	at java.net.SocketInputStream.socketRead(Unknown Source)
[2016-07-10 22:31:37] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-10 22:31:37] 	at java.net.SocketInputStream.read(Unknown Source)
[2016-07-10 22:31:37] 	at org.apache.http.impl.io.SessionInputBufferImpl.streamRead(SessionInputBufferImpl.java:136)
[2016-07-10 22:31:37] 	at org.apache.http.impl.io.SessionInputBufferImpl.fillBuffer(SessionInputBufferImpl.java:152)
[2016-07-10 22:31:37] 	at org.apache.http.impl.io.SessionInputBufferImpl.readLine(SessionInputBufferImpl.java:270)
[2016-07-10 22:31:37] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:140)
[2016-07-10 22:31:37] 	at org.apache.http.impl.conn.DefaultHttpResponseParser.parseHead(DefaultHttpResponseParser.java:57)
[2016-07-10 22:31:37] 	at org.apache.http.impl.io.AbstractMessageParser.parse(AbstractMessageParser.java:260)
[2016-07-10 22:31:37] 	at org.apache.http.impl.DefaultBHttpClientConnection.receiveResponseHeader(DefaultBHttpClientConnection.java:161)
[2016-07-10 22:31:37] 	at org.apache.http.impl.conn.CPoolProxy.receiveResponseHeader(CPoolProxy.java:153)
[2016-07-10 22:31:37] 	at org.apache.http.protocol.HttpRequestExecutor.doReceiveResponse(HttpRequestExecutor.java:271)
[2016-07-10 22:31:37] 	at org.apache.http.protocol.HttpRequestExecutor.execute(HttpRequestExecutor.java:123)
[2016-07-10 22:31:37] 	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:254)
[2016-07-10 22:31:37] 	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:195)
[2016-07-10 22:31:37] 	at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:86)
[2016-07-10 22:31:37] 	at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:108)
[2016-07-10 22:31:37] 	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184)
[2016-07-10 22:31:37] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82)
[2016-07-10 22:31:37] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:106)
[2016-07-10 22:31:37] 	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:57)
[2016-07-10 22:31:37] 	at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:138)
[2016-07-10 22:31:37] 	... 9 more
[2016-07-10 23:02:53] java.lang.IndexOutOfBoundsException: Index: 6, Size: 5
[2016-07-10 23:02:53] 	at java.util.LinkedList.checkElementIndex(Unknown Source)
[2016-07-10 23:02:53] 	at java.util.LinkedList.get(Unknown Source)
[2016-07-10 23:02:53] 	at com.vencillio.rs2.entity.player.net.in.impl.InterfaceAction.handle(InterfaceAction.java:88)
[2016-07-10 23:02:53] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 23:02:53] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 23:02:53] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 23:02:53] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 23:02:53] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
[2016-07-10 23:02:53] java.lang.IndexOutOfBoundsException: Index: 6, Size: 5
[2016-07-10 23:02:53] 	at java.util.LinkedList.checkElementIndex(Unknown Source)
[2016-07-10 23:02:53] 	at java.util.LinkedList.get(Unknown Source)
[2016-07-10 23:02:53] 	at com.vencillio.rs2.entity.player.net.in.impl.InterfaceAction.handle(InterfaceAction.java:88)
[2016-07-10 23:02:53] 	at com.vencillio.rs2.entity.player.net.in.PacketHandler.handlePacket(PacketHandler.java:151)
[2016-07-10 23:02:53] 	at com.vencillio.rs2.entity.player.net.Client.processIncomingPackets(Client.java:211)
[2016-07-10 23:02:53] 	at com.vencillio.rs2.entity.World.process(World.java:320)
[2016-07-10 23:02:53] 	at com.vencillio.core.GameThread.cycle(GameThread.java:124)
[2016-07-10 23:02:53] 	at com.vencillio.core.GameThread.run(GameThread.java:139)
